dbt_subprojects/solana/models/solana_utils/solana_utils_token_account_raw_data.sql

```diff
@@ -0,0 +1,128 @@
+{{
+  config (
+    schema='solana_utils',
+    alias='token_account_raw_data',
+    partition_by=['block_date'],
```

Comment:
I think we need this event feed to be partitioned by token_prefix so the main model has any chance of doing the lookups efficiently - in this setup trino would need to scan 100s of partitions to do the lead() and max() over window lookups, which is by far the most computationally expensive step in this pipeline